src/__tests__/jest/actions/authUser.test.js

```diff
@@ -15,10 +15,12 @@ import {
   signupError
 } from "../../../redux/actions/authUser";
 
+import config from "../../../config";
+
 const middlewares = [thunk];
 const mockStore = configureMockStore(middlewares);
 let store;
-const signupUrl = "http://127.0.0.1:8000/api/users/signup/";
+const signupUrl = `${config}/users/signup/`;
```

Comment:
@PAbishai shouldn't it be `${config.BASE_URL}`

Reply:
```suggestion
const signupUrl = `${config}/users/signup/`;
```
Besides this small change, I think everything is working as expected.

Reply:
@PAbishai response to @Bileonaire 's question?

Reply:
@Bileonaire you are right. Amending that.